metacat-main/src/main/java/com/netflix/metacat/main/api/RequestWrapper.java

```diff
@@ -87,7 +98,11 @@ public RequestWrapper(@NotNull @NonNull final Registry registry) {
      */
     public QualifiedName qualifyName(final Supplier<QualifiedName> nameSupplier) {
         try {
-            return nameSupplier.get();
+            final QualifiedName name = nameSupplier.get();
+            if (config.isTableAliasEnabled() && name.isTableDefinition()) {
```

Comment:
Why not put the ```name.isTableDefinition()``` check inside the ```aliasService.getTableName()``` API?  The ```getTableName()``` API indicates to me that it's only for tableNames not database, view, or catelog names. It makes sense to me to let the API handle the ```name.isTableDefinition()``` check.  If the API is something like ```getAlias()``` and works for all QualifiedName, check the tableName outside seems a way to control only return the Aliasname for TableName.

Reply:
That makes sense however the `qualifyName(nameSupplier)` API is called for all QualifiedNames, i.e `nameSupplier.get()` could return a Catalog, Database, Table, Partition or View name. Since we only want to check alias for table names, I figured an explicit check in this method would make more sense instead of calling the `aliasService.getTableName()` for every type of QualifiedName.